tests/unit/services/cloudformation/test_deployment_utils.py

```diff
@@ -0,0 +1,27 @@
+from localstack.services.cloudformation.deployment_utils import fix_boto_parameters_based_on_report
+from localstack.testing.pytest import markers
+
+
+class TestFixBotoParametersBasedOnReport:
+    @markers.aws.only_localstack
```

Comment:
```suggestion
```
these are only required for tests that can potentially be executed against AWS (anything in `tests/aws/`)

Reply:
Ohh, I think I tried writing a test there and saw the "required message" so just mindlessly copied the markers over. Good to know. I'll remove them.